src/test/java/core/basesyntax/DataParserServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParserService;
+import core.basesyntax.service.impl.DataParserServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DataParserServiceTest {
+    private static List<FruitTransaction> excepted;
+    private static List<String> input;
+    private static DataParserService parserService;
+    private static final String FIRST_FRUIT_NAME = "durian";
+    private static final String SECOND_FRUIT_NAME = "papaya";
+    private static final int FIRST_QUANTITY = 100;
+    private static final int SECOND_QUANTITY = 55;
+    private static final int THIRD_QUANTITY = 28;
+    private static final int FOURTH_QUANTITY = 45;
+
+    @BeforeClass
+    public static void createOutputList() throws Exception {
+        parserService = new DataParserServiceImpl();
+        excepted = new ArrayList<>();
+        FruitTransaction first = new FruitTransaction();
+        first.setOperation(FruitTransaction.Operation.BALANCE);
+        first.setFruit(FIRST_FRUIT_NAME);
+        first.setQuantity(FIRST_QUANTITY);
+        excepted.add(first);
+        FruitTransaction second = new FruitTransaction();
+        second.setOperation(FruitTransaction.Operation.BALANCE);
+        second.setFruit(SECOND_FRUIT_NAME);
+        second.setQuantity(SECOND_QUANTITY);
+        excepted.add(second);
+        FruitTransaction third = new FruitTransaction();
+        third.setOperation(FruitTransaction.Operation.PURCHASE);
+        third.setFruit(FIRST_FRUIT_NAME);
+        third.setQuantity(THIRD_QUANTITY);
+        excepted.add(third);
+        FruitTransaction fourth = new FruitTransaction();
+        fourth.setOperation(FruitTransaction.Operation.SUPPLY);
+        fourth.setFruit(SECOND_FRUIT_NAME);
+        fourth.setQuantity(FOURTH_QUANTITY);
+    }
+
+    @Before
+    public void createInputList() throws Exception {
```

Comment:
```suggestion
    public void createInputList() {
```
same everywhere where needed